.github/workflows/build-toolchain.yml

```
@@ -662,17 +827,17 @@ jobs:
 
   release:
     runs-on: ubuntu-latest
-    needs: [context, windows-build]
+    needs: [context, windows-build-x64]
```

This should be dependent on both of the architecture builds.

---

Ok. `windows-build-arm64` currently fails. Do you know of a way to make it continue on an arm64 failure?

---

Added `continue-on-error: true` to `windows-build`.

---

Used `if success() || failure()` on `release` instead of `continue-on-error` as it didn't work.